docs/core-concepts/bot-sequences/actions.mdx

```diff
@@ -1,9 +1,624 @@
+`type` is set to `MonkeyBot`.
+
+```json
+// The `botAction.data` format
+{
+    "actionInterval": ... // The interval between actions in seconds,
+    "actionSettings": {
+        "DisabledActionPaths": [
+            ... // An optional list of strings of the format "Game Object Path/Method Path" to disable from the monkey bot
+        ]
+    }
+}
+```
+
+**Example: Running a monkey bot to click on random objects while ignoring the quit actions**
+
+```json
+{
+    "name": "Run monkey bot",
+    "description": "Runs a monkey bot to click on random objects while ignoring the quit actions",
+    "endCriteria": [...],
+    "botAction": {
+        "type": "MonkeyBot",
+        "data": {
+            "actionInterval": 0.5,
+            "actionSettings": {
+                "DisabledActionPaths": [
+                    "Unity UI/Button/Unity.Multiplayer.Samples.BossRoom.Visual.UISettingsCanvas.OnClickQuitButton",
+                    "Unity UI/Button/Unity.Multiplayer.Samples.BossRoom.Debug.DebugCheatsManager.GoToPostGame"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Mouse_CVImage
+
+This action type clicks on a region of the screen based on a template image found using computer vision. This is useful for
+clicking on certain icons or objects that may have dynamic positions or are changing often during development.
+
+**Data Format**
+
+`type` is set to `Mouse_CVImage`.
+
+- `imageData` - The image data in one of the following formats...
+    - The base64 encoded string of the JPG image data - This must be the entire JPG image file, not just the visible bytes.
+    - A file:// path to a JPG or PNG image - Be careful when using relative paths as these will be interpreted differently in the Editor vs Runtime builds.
+    - A resource:// path to a READABLE Texture2D in one of your project's Resources folders.
+    - A resource:// path to a .bytes TextAsset in one of your project's Resources folders that is a JPG or PNG file saved with a .bytes file extension.  This can be used if you do not want Unity to import your image as a Texture.
+- `withinRect` - An optional (can be null/undefined) field to limit the search area to a specific pixel region of the current frame.  The SDK will linearly tranform the supplied `rect` to fit the current resolution using the `screenSize` as the initial reference resolution.
+    - `screenSize` - The reference resolution in pixels which defines the screen space that `rect` is defined within.
+    - `rect` - The position (x=0, y=0 is bottom left) and size (width, height) of the rectangle that must contain the supplied image data.  The values are defined in pixels.
+- `actions` - The list of mouse actions to take at the center point of the returned `rect`.
+
+**Notes**
+
+The `CVImage` type is still in an experimental phase and may provide inconsistent or unexpected results in many situations.
+- Multiple matches of the specified image within a frame may provide incorrect or inconsistent result bounds.
```

Comment:
We need to add a blurb about how images are compared in grey scale

Reply:
Added